src/modules/iotjs_module_udp.c

```diff
@@ -283,61 +283,44 @@ JS_FUNCTION(GetSockeName) {
 }
 
 
-#define IOTJS_UV_SET_SOCKOPT(fn)                          \
-  JS_DECLARE_THIS_PTR(udpwrap, udp_wrap);                 \
-  DJS_CHECK_ARGS(1, number);                              \
-                                                          \
-  int flag = JS_GET_ARG(0, number);                       \
-  int err = fn(iotjs_udpwrap_udp_handle(udp_wrap), flag); \
-                                                          \
-  return jerry_create_number(err);
-
-
-JS_FUNCTION(SetBroadcast) {
-#if !defined(__NUTTX__)
-  IOTJS_UV_SET_SOCKOPT(uv_udp_set_broadcast);
-#else
-  IOTJS_ASSERT(!"Not implemented");
-
-  return jerry_create_null();
-#endif
-}
-
-
-JS_FUNCTION(SetTTL) {
-#if !defined(__NUTTX__)
-  IOTJS_UV_SET_SOCKOPT(uv_udp_set_ttl);
-#else
-  IOTJS_ASSERT(!"Not implemented");
-
-  return jerry_create_null();
-#endif
-}
-
+enum config_type { BROADCAST, TTL, MULTICASTTTL, MULTICASTLOOPBACK };
```

Comment:
We should add a comment that the order should match the dgram.js order. A comment should be here and in the dgram.js so if someone updates these then it is clear where should the developer look further.